kasigazibrian/WeConnectReact
# #158386207 Edit ReadMe file

## What does this PR do?
Edits the readme file

## Description of Task to be completed?
Edits the readme and adds information about the application
and how to set it up to run on your machine.

## How should this be manually tested?
Navigate to the application repository and you should be able to view the readme with the information
about the application and how to set it up

<img width="983" alt="screen shot 2018-07-04 at 13 26 37" src="https://user-images.githubusercontent.com/26734746/42272183-09ab9040-7f8e-11e8-9c90-b6bdb49bc680.png">
<img width="982" alt="screen shot 2018-07-04 at 13 26 54" src="https://user-images.githubusercontent.com/26734746/42272194-12907f4a-7f8e-11e8-82aa-72c52ac2369b.png">
<img width="982" alt="screen shot 2018-07-04 at 13 27 10" src="https://user-images.githubusercontent.com/26734746/42272199-172c3684-7f8e-11e8-95ce-c68c5472f239.png">

 ## Any background context you want to provide?
Previously, the readme file was not edited.

## What are the relevant pivotal tracker stories?
[#158386207](https://www.pivotaltracker.com/story/show/158386207)
